### ISSUE TITLE
Deleted devices reappear

### ISSUE THREAD

**Issue by MaleNurse** [pucherot/Pi.Alert#84]
Not sure if this had anything to do with #80 or #83, but devices that have been deleted and are not not the network or in ARP reappear after the next 1min scan as new devices with the same MAC and IP addresses.

Logs: 
[logs.zip](https://github.com/pucherot/Pi.Alert/files/5967023/logs.zip)

Please let me know if there is any more info that may be useful to you.

**Comment by eugef66** [comment]
same here. I had a guest who i connected to my wifi and after he left I deleted device form the list, but it's coming back aftre every run. I assume it's coming from pihole FTL database. I used pihole shell command to flash network table and cache, but the device is still poping back up. Only after I restarted pihole completely the device disappeared from FTL database and deleted devices in pialert is no longer appearing. 

I think there should be an option to "archive" device so it doesn't appear in the list anymore and all archived devices should get automatically deleted aftre some time. Of cause there should be an option to see archived devices list when needed and "unarchived"

Again, thank you for a great work.
